airgun/widgets.py

```diff
@@ -1174,7 +1174,7 @@ def __init__(self, parent, id=None, locator=None, logger=None):
 
     def read(self):
         """Return drop-down selected item value"""
```

Comment:
@amolpati30 Thanks for adding this change. Can you please add a small comment above this line so that anyone who sees the reason of adding it? This is a non-blocker comment.